catalog-info.yaml

```diff
@@ -21,7 +21,7 @@ metadata:
   description: 'Pipeline for the stream project'
   links:
     - title: Pipeline
-      url: https://buildkite.com/elastic/stream-pub-sub
+      url: https://buildkite.com/elastic/stream
```

Comment:
while at it, can we update the above schema in line 16 to https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json so that it reflects the latest version?

Details in this [internal link](https://elastic.slack.com/archives/C0522G6FBNE/p1700477179859799).